### DISCUSSION TITLE
Is the service down?

### DISCUSSION THREAD

**Discussion by RaphaCastelloes** [community/community#6970] [Copilot Conversations]
That´s the question... By the way, what a great extension! I´m addicted to!

**Comment by Robbie-Cook** [comment]
Yeah, not working for me either

**Reply by RaphaCastelloes** [reply]
Gosh. What a shame. Is it usual?

**Reply by TarunavBA** [reply]
@Robbie-Cook same for me too, @RaphaCastelloes dunno.

**Comment by davecheney** [comment]
@Robbie-Cook can you please paste the output from View -> Output; choose Github Copilot from the pulldown.

**Reply by RaphaCastelloes** [reply]
Hi @davecheney , I don´t know if there is any specific setup in his machine, althought I´m sending mine output.

[INFO] [ghostText] [2021-10-29T03:11:59.917Z] Cancelled after extractPrompt
[INFO] [default] [2021-10-29T03:12:00.018Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:12:00.440Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:12:00.566Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:12:00.604Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:12:00.740Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:12:01.293Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:13:14.686Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:13:15.789Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:13:15.936Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:13:15.977Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:13:16.106Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-10-29T03:13:26.243Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 10136 ms
[ERROR] [streamChoices] [2021-10-29T03:13:26.251Z] Unhandled status from server:,504,{"error":{"message":"request timeout"}}

[INFO] [ghostText] [2021-10-29T03:13:26.253Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:13:28.072Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-10-29T03:13:38.210Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 10137 ms
[ERROR] [streamChoices] [2021-10-29T03:13:38.214Z] Unhandled status from server:,504,{"error":{"message":"request timeout"}}

[INFO] [ghostText] [2021-10-29T03:13:38.215Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:28:06.718Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-10-29T03:28:07.800Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2021-10-29T03:28:07.939Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 1220 ms
[INFO] [streamChoices] [2021-10-29T03:28:07.941Z] Cancelled by server
[INFO] [ghostText] [2021-10-29T03:28:07.941Z] Breaking, no choices
[INFO] [ghostText] [2021-10-29T03:28:07.945Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:28:08.096Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:28:12.838Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:28:12.991Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:28:13.010Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:28:13.131Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:28:13.268Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:28:13.411Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:28:19.552Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:29:14.608Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:29:18.243Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:29:50.661Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:29:51.190Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:29:51.316Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:29:52.402Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:31:20.624Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:31:20.988Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:31:21.106Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [ghostText] [2021-10-29T03:31:21.129Z] Breaking, no choices
[INFO] [default] [2021-10-29T03:31:21.261Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex

**Reply by davecheney** [reply]
@RaphaCastelloes thanks for that data, I can confirm that copilot is working and there are no problems in any of our data centers at the moment.

**Reply by RaphaCastelloes** [reply]
@davecheney, you are welcome my friend, maybe is something in my enviroment. @Robbie-Cook, please, can you confirm if it is still down for you? For me, it is not working anymore.

**Reply by Robbie-Cook** [reply]
@RaphaCastelloes Sorry mate, back up for me, sorry for late reply
